### PR TITLE
Separating function update actions.

### LINKED ISSUE #416: Separate actions for update lambda code and update lambda configuration
Two different right-click context menu options that load two different screens.

### PR DESCRIPTION
**Pull request by kiiadi**
Fixes #416

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Description

Based on design review feedback, separating out "update function configuration" and "update function code" as this more closely matches the Lambda service [API](https://docs.aws.amazon.com/lambda/latest/dg/API_Operations.html) and should be less surprising for customers.

## Motivation and Context

More consistent with Lambda's APIs should make it clearer what is actually going on.

## Related Issue(s)

#416 #430

### Create dialog
screen shot 2018-11-10 at 5 01 02 pm

### New menu
screen shot 2018-11-10 at 5 01 14 pm

### Update Configuration dialog
screen shot 2018-11-10 at 5 01 22 pm

### Update Code dialog
screen shot 2018-11-10 at 5 01 33 pm

### Disabled 'Update code' when source not found
screen shot 2018-11-10 at 5 08 55 pm

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [x] Local run of `gradlew check` succeeds
- [x] My code follows the code style of this project
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed
- [ ] A short description of the change has been added to the **CHANGELOG**

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by abrooksv** [comment]
Still not a fan of the update code dialog, if you rename the class or the handler, we will be unable to upload the code anymore.

**Comment by kiiadi** [comment]
@abrooksv - that's a great point. So at a minimum we'd need to include the handler - I think there is value in having a separate looking screen so that customers know the difference though.

**Comment by abrooksv** [comment]
Fair enough compromise.

**Comment by abrooksv** [comment]
Oh, and we always have to have the action to update code enabled, else the user will have to run Update Configuration, before Update Code is enabled if they rename/move the handler around

**Comment by kiiadi** [comment]
@abrooksv will do the handler thing as part of this #430

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/upload/EditFunctionPanel.java [hunk 1]
```diff
@@ -35,13 +35,15 @@
...
+    @NotNull JPanel configurationSettings;
 
     private final Project project;
 
-    EditLambdaPanel(Project project) {
+    EditFunctionPanel(Project project) {
         this.project = project;
 
         deploySettings.setBorder(IdeBorderFactory.createTitledBorder("Deployment Settings", false));
+        configurationSettings.setBorder(IdeBorderFactory.createTitledBorder("Configuration Settings", false));
```
Comment (abrooksv) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/upload/EditFunctionPanel.java]: +1 yay, but move in to messages()

**Comment by kiiadi** [comment]
Updated to show the handler on update code like so:
screen shot 2018-11-12 at 1 46 39 pm

**Pull request merged**